### PR TITLE
Add documentation for multi select and use of commas in allowed values

### PR DESCRIPTION
**Pull request by shalini-agr**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by archit-harness — DISMISSED** [review]

**Review by archit-harness — DISMISSED** [review]

File: docs/platform/20_References/runtime-inputs.md [hunk 1]
```diff
@@ -71,6 +71,20 @@ You can use Runtime Inputs in a CI stage's Infrastructure. Here's an example usi
 You can use Runtime Inputs for the Service in a CD stage's Service settings.
 
 ![](./static/runtime-inputs-06.png)
+
+### Using allowed and default values in runtime inputs
+
+You can add allowed values and a default value for a runtime input. Default value should be present in the list of allowed values.
```
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:77]: Do customers know what are the allowed and default values? Do we have that listed somewhere?

File: docs/platform/20_References/runtime-inputs.md [hunk 2]
```diff
@@ -71,6 +71,20 @@ You can use Runtime Inputs in a CI stage's Infrastructure. Here's an example usi
 You can use Runtime Inputs for the Service in a CD stage's Service settings.
 
 ![](./static/runtime-inputs-06.png)
+
+### Using allowed and default values in runtime inputs
+
+You can add allowed values and a default value for a runtime input. Default value should be present in the list of allowed values. 
+For runtime inputs defined for pipeline, stage, and shell script variables, multi selection is allowed. 
+Multi Selection is currently behind the feature flag `PIE_MULTISELECT_AND_COMMA_IN_ALLOWED_VALUES`
```
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:79]:
```suggestion
The multiple selection functionality is currently behind the feature flag, `PIE_MULTISELECT_AND_COMMA_IN_ALLOWED_VALUES`
```

File: docs/platform/20_References/runtime-inputs.md [hunk 3]
```diff
@@ -71,6 +71,20 @@ You can use Runtime Inputs in a CI stage's Infrastructure. Here's an example usi
...
+
+![](./static/runtime-inputs-11.png)
+
+### Using commas in allowed and default values in runtime inputs
+
+Use of comma is supported in allowed values and default value. The string with comma needs to be wrapped within `\'   \'`
```
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:85]:
```suggestion
Commas are supported in both allowed and default values. A string with comma must be wrapped within `\'   \'`. 
```

File: docs/platform/20_References/runtime-inputs.md [hunk 4]
```diff
@@ -71,6 +71,20 @@ You can use Runtime Inputs in a CI stage's Infrastructure. Here's an example usi
...
+You can add allowed values and a default value for a runtime input. Default value should be present in the list of allowed values. 
+For runtime inputs defined for pipeline, stage, and shell script variables, multi selection is allowed. 
+Multi Selection is currently behind the feature flag `PIE_MULTISELECT_AND_COMMA_IN_ALLOWED_VALUES`
+
+![](./static/runtime-inputs-11.png)
+
+### Using commas in allowed and default values in runtime inputs
+
+Use of comma is supported in allowed values and default value. The string with comma needs to be wrapped within `\'   \'` 
+For Eg: `<+input>.default(\'london,uk\').allowedValues(\'bengaluru,india\',\'newyork,usa\',\'london,uk\')`
```
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:86]:
```suggestion
For example, `<+input>.default(\'london,uk\').allowedValues(\'bengaluru,india\',\'newyork,usa\',\'london,uk\')`.
```

**Review by roshnisarangadharan — DISMISSED** [review]

File: docs/platform/20_References/runtime-inputs.md (hunk 1 above)
Comment (SudarshanPurohit) [inline docs/platform/20_References/runtime-inputs.md:77]: I think this should be "You can *set* allowed values and a default value for a runtime input.

**Review by SudarshanPurohit — DISMISSED** [review]
Looks good to me, with one suggestion to clarify.

File: docs/platform/20_References/runtime-inputs.md (hunk 1 above)
Comment (shalini-agr) [reply docs/platform/20_References/runtime-inputs.md:77]: We have the option to add allowed and default values for almost all of the fields that can be runtime input. So I think customers would know about these.

File: docs/platform/20_References/runtime-inputs.md [hunk 5]
```diff
@@ -71,6 +71,20 @@ You can use Runtime Inputs in a CI stage's Infrastructure. Here's an example usi
 You can use Runtime Inputs for the Service in a CD stage's Service settings.
 
 ![](./static/runtime-inputs-06.png)
+
+### Using allowed and default values in runtime inputs
```
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:75]:
```suggestion
### Using allowed values and default values in runtime inputs
```

File: docs/platform/20_References/runtime-inputs.md (hunk 1 above)
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:77]:
```suggestion
You can set allowed values and a default value for a runtime input. The default value should be present in the list of allowed values. 
```

Comment (roshnisarangadharan) [reply docs/platform/20_References/runtime-inputs.md:77]: I'm assuming that you can set all allowed values and one default value for a runtime input. If not then we should say ...set allowed values and default values..

**Review by roshnisarangadharan — DISMISSED** [review]

File: docs/platform/20_References/runtime-inputs.md [hunk 6]
```diff
@@ -71,6 +71,20 @@ You can use Runtime Inputs in a CI stage's Infrastructure. Here's an example usi
 You can use Runtime Inputs for the Service in a CD stage's Service settings.
 
 ![](./static/runtime-inputs-06.png)
+
+### Using allowed values and default values in runtime inputs
+
+You can set allowed values and a default value for a runtime input. The default value should be present in the list of allowed values. 
+For runtime inputs defined for pipelines, stages, and shell script variables, multiple selections are allowed. 
+The multiple selection functionality is currently behind the feature flag, `PIE_MULTISELECT_AND_COMMA_IN_ALLOWED_VALUES`
```
Comment (roshnisarangadharan) [inline docs/platform/20_References/runtime-inputs.md:79]:
```suggestion
The multiple selection functionality is currently behind the feature flag, `PIE_MULTISELECT_AND_COMMA_IN_ALLOWED_VALUES`.
```

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**